### PR TITLE
Blackstone map

### PR DESCRIPTION
**Pull request by Real-MAGNUM**
## About The Pull Request

Merges the bog and town into one map, blackstone.dmm.
Restructures the bogtown to not be as spaced out.
Moves dwarves/the mines back to their ORIGINAL spot to the west of the blacksmiths.
Fixes a lot of bugs including smithy roof.
More water/rivers that will be expanded on with cave systems.
Preperation for dungeons at the adventurer camp (the tents) in bogtown.
Fixes day/night lighting for the entire map.

## Why It's Good For The Game

Lighting was stupid and so are travel tiles when used outside of antag spawns.

### CODE REVIEW

**Comment by Real-MAGNUM** [comment]
Headdev approves testmerging

**Comment by zakanater19** [comment]
peak

**Comment by Real-MAGNUM** [comment]
If nothing is reported next round it's ready for fullmerge.

**Comment by Knightfall5** [comment]
The new (old) mines suck. A single pickaxe got me 8 coal, 5 iron. The volcano mines give me easily double this on a bad day, sometimes I've gotten 30 various ores from a single pickaxe. At the very least put back the mountain pass in so getting to and from the volcano isn't a chore since it is the only worthwhile place to mine still.

Also all the old mining supplies are gone, so now we have to suffer through waiting at the smithy roundstart trying to get a pickaxe, and by the time you get through that you could have already been done mining. We already had a severe lack of people mining, the smithies shouldn't be dependent on the steward importing everything they need or being stuck without any materials all round.

And why was all the water in town changed to sewage? The only two wells in town are behind locked gates at roundstart, making getting water more of a hassle than it should be.

tl;dr, old map better. Only good change is the removal of the travel points.

**Comment by Real-MAGNUM** [comment]
Old volcano still exists, it just requires you to go into the bog to reach. Enter the lake area from top z level, you will see the stairwell. Risk for reward, though if you go deep enough into the "old" mines, you will find the same tier rocks.

**Comment by Knightfall5** [comment]
If you think I'm going to stand around begging the smithies for a pickaxe for ten minutes then spend even more time mining just to get to the actually good rocks/trek through the bog to the volcano you are crazy. I just won't mine anymore, simple as that. And judging by the distinct lack of miners even before this change, so will everyone else.
